client/dist/bundle.js

```diff
@@ -94,7 +94,7 @@
 subClass.prototype = Object.create(superClass && superClass.prototype, { constructor: { value: subClass, writable: true, configurable: true } }); if (superClass) _setPrototypeOf(subClass, superClass); }\n\nfunction _setPrototypeOf(o, p) { _setPrototypeOf = Object.setPrototypeOf || function _setPrototypeOf(o, p) { o.__proto__ = p; return o; }; return _setPrototypeOf(o, p); }\n\n\n\nvar Comment =\n/*#__PURE__*/\nfunction (_React$Component) {\n  _inherits(Comment, _React$Component);\n\n  function Comment(props) {\n    _classCallCheck(this, Comment);\n\n    return _possibleConstructorReturn(this, _getPrototypeOf(Comment).call(this, props));\n  }\n\n  _createClass(Comment, [{\n    key: \"render\",\n    value: function render() {\n      return react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(\"div\", {\n        className: \"commentBlock\"\n      }, react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(\"div\", {\n        className: \"pic\"\n      }, react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(\"img\", {\n        src: this.props.user.profileImageURL,\n        height: \"28\",\n        width: \"28\"\n      })), react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(\"div\", {\n        className: \"words comment\"\n      }, react__WEBPACK_IMPORTED_MODULE_0___default.a.createElement(\"a\", {\n        className: \"username\",\n        href: \"\"\n      }, this.props.user.username), this.props.user.comment));\n    }\n  }]);\n\n  return Comment;\n}(react__WEBPACK_IMPORTED_MODULE_0___default.a.Component);\n\n/* harmony default export */ __webpack_exports__[\"default\"] = (Comment);//# sourceURL=[module]\n//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiLi9jbGllbnQvc3JjL2NvbXBvbmVudHMvQ29tbWVudC5qc3guanMiLCJzb3VyY2VzIjpbIndlYnBhY2s6Ly8vLi9jbGllbnQvc3JjL2NvbXBvbmVudHMvQ29tbWVudC5qc3g/YjJlNSJdLCJzb3VyY2VzQ29udGVudCI6WyJpbXBvcnQgUmVhY3QgZnJvbSAncmVhY3QnO1xuXG5jbGFzcyBDb21tZW50IGV4dGVuZHMgUmVhY3QuQ29tcG9uZW50IHtcbiAgY29uc3RydWN0b3IocHJvcHMpIHtcbiAgICBzdXBlcihwcm9wcylcbiAgfVxuXG4gIHJlbmRlcigpIHtcbiAgICByZXR1cm4gKFxuICAgICAgPGRpdiBjbGFzc05hbWU9XCJjb21tZW50QmxvY2tcIj5cbiAgICAgICAgPGRpdiBjbGFzc05hbWU9XCJwaWNcIj5cbiAgICAgICAgICA8aW1nIHNyYz17dGhpcy5wcm9wcy51c2VyLnByb2ZpbGVJbWFnZVVSTH0gaGVpZ2h0PVwiMjhcIiB3aWR0aD1cIjI4XCI+PC9pbWc+XG4gICAgICAgIDwvZGl2PlxuICAgICAgICA8ZGl2IGNsYXNzTmFtZT1cIndvcmRzIGNvbW1lbnRcIj5cbiAgICAgICAgICA8YSBjbGFzc05hbWU9XCJ1c2VybmFtZVwiIGhyZWY9XCJcIj57dGhpcy5wcm9wcy51c2VyLnVzZXJuYW1lfTwvYT5cbiAgICAgICAgICB7dGhpcy5wcm9wcy51c2VyLmNvbW1lbnR9XG4gICAgICAgIDwvZGl2PlxuICAgICAgPC9kaXY+XG4gICAgKTtcbiAgfVxufVxuXG5leHBvcnQgZGVmYXVsdCBDb21tZW50O1xuIl0sIm1hcHBpbmdzIjoiOzs7Ozs7Ozs7Ozs7Ozs7Ozs7Ozs7QUFBQTtBQUNBO0FBQ0E7Ozs7O0FBQ0E7QUFBQTtBQUNBO0FBREE7QUFFQTtBQUNBOzs7QUFDQTtBQUNBO0FBQ0E7QUFBQTtBQUNBO0FBQUE7QUFDQTtBQUFBO0FBQUE7QUFBQTtBQUVBO0FBQUE7QUFDQTtBQUFBO0FBQUE7QUFLQTs7OztBQWpCQTtBQUNBO0FBbUJBIiwic291cmNlUm9vdCI6IiJ9\n//# sourceURL=webpack-internal:///./client/src/components/Comment.jsx\n");
```

Comment:
Wouldn't it be better to gitignore this file?

Reply:
Yes, it would be. Not sure why this got pushed up since I have in my .gitignore file. I'll double check to make sure. Good catch!